<html>
<head>
    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, user-scalable=no" />
<meta http-equiv="content-language" content="en" />
<meta http-equiv="content-type" content="Type=text/html; charset=utf-8" />
<link rel="shortcut icon" type="image/x-icon" href="/assets/favicon.ico"/>

<meta name="description" content="Personal site of Augustin Luna">

<meta name="google-site-verification" content="VAGpLrwLLA8WizqfUum5tXY5kjLTz7OD0d_ST5JzxSo" />
<meta name="msvalidate.01" content="7EB565E2F88773ACBD0A245005E17D24" />

<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" />
<!--<link rel="stylesheet" href="/assets/font-awesome-4.6.3/css/font-awesome.min.css" />-->

<link rel="stylesheet" href="/assets/css/main.css" />
<link rel="stylesheet" href="/assets/academicons-1.7.0/css/academicons.css" />

<!--<script type="text/javascript" src='/assets/js/bootstrap.min.js'></script>-->
<!--<script type="text/javascript" src='/assets/js/jquery-1.12.4.min.js'></script>-->

<script type="text/javascript" src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
<script type="text/javascript" src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<title>Augustin Luna - Lunean</title>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-317478-11', 'auto');
  ga('send', 'pageview');
</script>


</head>

<body>
  <div id="landing" class="container">
    <div id="landing-left" class="col-lg-5">
        <div class="affix">
              <h1>Augustin Luna</h1>
  <h2 class="headline">Research Fellow | Department of Biostatistics and Computational Biology, Dana-Farber Cancer Institute</h2>

  <div id="landing-entry">
      <h4>aluna AT jimmy.harvard.edu</h4>
  </div>
  <div id="landing-list">
      <ul>
          <li><a id="linkedin" class="landingListEntry" href="https://www.linkedin.com/in/augustin-luna-12473510" target="_blank"><i class="fa fa-linkedin-square"></i> LinkedIn</a></li>
          <li><a id="github" class="landingListEntry" href="https://github.com/cannin" target="_blank"><i class="fa fa-github-square"></i> GitHub</a></li>
          <li><a id="twitter" class="landingListEntry" href="https://twitter.com/canninl" target="_blank"><i class="fa fa-twitter-square"></i> Twitter</a></li>
          <li><a id="google-scholar" class="landingListEntry" href="https://scholar.google.com/citations?user=u2dgLp8AAAAJ" target="_blank"><i class="ai ai-google-scholar-square"></i> Google Scholar</a></li>
          <li><a id="blog" class="landingListEntry" href="http://blog.lunean.com" target="_blank"><i class="fa fa-pencil-square"></i> Blog</a></li>
          <li>&nbsp;</li>
          <li><a id="random-walks" class="landingListEntry" href="http://randomwalks.lunean.com" target="_blank"><i id="redSquare" class="fa fa-square"></i> Random Walks Exhibit</a></li>
      </ul>
  </div>

        </div>
    </div>
    <div id="landing-right" class="col-lg-7">
        <h1>Experiences</h1>

    <h2>Post-Baccalaureate Intramural Research Training Award Fellow / National Institute of Mental Health (10/2005-06/2007)</h2>
    <p>Worked with Daniel Weinberger and Kristen K. Nicodemus. Wrote an automated statistical analysis pipeline for candidate gene studies. Wrote software to visualize the results of SNP analyses. Performed analysis in a simulation study assessing power and type I error of single SNP transmission disequilibrium tests (TDTs)
</p>

    <h2>Pre-Doctoral Cancer Research Training Award Fellow / National Cancer Institute (07/2008-08/2013)</h2>
    <p>Worked with Mirit I. Aladjem and Kurt W. Kohn. Worked on standardized representations for biological data (Molecular Interaction Maps (MIMs), Systems Biology Graphical Notation (SBGN), and BioPAX) and related software for MIMs: XML-based format, API, graphical editor, validator, scripting interface, and faceted browser. Worked on an ODE-based model of interactions that connect components of the circadian clock to those of DNA damage response; collaborating with Geoffrey B. McFadden (at NIST). Mentored student projects through the Google Summer of Code Project (2011 and 2012). </p>

    <h2>Undergraduate Research Assistant / Georgia Institute of Technology (09/2004-05/2005)</h2>
    <p>Worked with Andreas Bommarius and Karen M. Polizzi. Created an expression vector for mutant variants in a high throughput screen. Collected information on antibiotic resistance in E.coli due to beta-lactamase
</p>

    <h2>Bioinformatics PhD Student / Boston University (08/2007-07/2013)</h2>
    <p>Worked with Daniel Segrè. Worked on a metabolic network to represent a kidney cell using BIGG human metabolic network and microarray data from the GEO repository for flux balance analysis. </p>

    <h2>Intern / Pfizer (06/2010-07/2010)</h2>
    <p>Worked with Jacob Glanville. Developed a wiki-based platform for the analysis and collaborative annotation of experimental results.</p>

    <h2>Research Fellow / Memorial Sloan-Kettering Cancer Center (09/2013-10/2015)</h2>
    <p>Worked with Chris Sander. Working on standardized representations of biological data (BioPAX), the aggregation of multiple pathway databases into Pathway Commons, and related software, including: Paxtools and paxtoolsR. In collaboration with the Yves Pommier lab of the Developmental Therapeutics Branch at the NCI, working to identify novel drug-target interactions using compound activity from the NCI Developmental Therapeutics Program (DTP) and genomic profiling data of the NCI-60 cancer cell lines. Compounds are prioritized for further development using understanding of pharmacology, target pathway context, relevance to cancer, and novelty. </p>

    <h2>Undergraduate Research Assistant / Georgia Institute of Technology (09/2003-05/2004)</h2>
    <p>Worked with Allen Tannenbaum and Eli Hershkovitz. Worked on the classification of RNA classification into motif libraries. Wrote programs to parse tRNA X-ray crystallography data
</p>

    <h2>Research Fellow / Dana-Farber Cancer Institute (11/2015-Present)</h2>
    <p>Work with Chris Sander. Develop R packages for cancer pharmacology research, including cgdsr (CBioPortal, TCGA studies), paxtoolsr (Pathway Commons), rcellminer (NCI-60 cell lines). Develop R based integrative analyses and bioinformatics pipelines with a focus on reproducibility using Docker. Develop web applications for cancer pharmacology databases and analyses. Work on tools to conduct complex pathway analyses using the aggregated Pathway Commons database. Work to identify novel drug-target interactions using compound activity from the NCI Developmental Therapeutics Program (DTP) and omic profiling data of the NCI-60 cancer cell lines in collaboration with the Developmental Therapeutics Branch at the NCI, the Genomics of Drug Sensitivity in Cancer Project at the Sanger Institute, and the Institute of Molecular Systems Biology at ETH Zurich. Work to identify novel drug combinations for cancer using proteomics data. </p>


<h1>Education</h1>

    <h4>Bioinformatics, PhD / Boston University / 2007-2013</h4>

    <h4>Biomedical Engineering, BS / Georgia Institute of Technology / 2002-2005</h4>

    <h4>Chemistry, AS / Middle Georgia College / 2000-2002</h4>


<h1>Honors</h1>

    <h4>Petit Undergraduate Research Scholar / 2004</h4>

    <h4>Postdoctoral Ruth L. Kirschstein National Research Service Award / 2015</h4>

    <h4>NIH Post-Baccalaureate Intramural Research Training Award / 2005</h4>

    <h4>Goizueta Foundation Scholarship / 2004</h4>

    <h4>Alpha Eta Mu Beta Biomedical Engineering Honor Society / 2005</h4>

    <h4>Georgia Governor’s Scholar / 2001</h4>

    <h4>Tau Beta Pi Engineering Honor Society / 2005</h4>

    <h4>NCI Pre-Doctoral Cancer Research Training Award / 2008</h4>

    <h4>Golden Key Academic Honor Society / 2005</h4>

    <h4>HOPE Scholarship / 2001</h4>

    <h4>Ford Foundation Dissertation Fellowship / 2012</h4>


<h1>Publications</h1>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/21636591">PathVisio-MIM: PathVisio plugin for creating and editing Molecular Interaction Maps (MIMs). Bioinformatics (Oxford, England) (2011)</a></h4>

        <!--
        <p><a class="toggle" title="PathVisio-MIM: PathVisio plugin for creating and editing Molecular Interaction Maps (MIMs)." href="#">(Show/Hide Details)</a></p>

        <div id="1" style="display: none">
            <p>MOTIVATION: A plugin for the Java-based PathVisio pathway editor has been developed to help users draw diagrams of bioregulatory networks according to the Molecular Interaction Map (MIM) notation. Together with the core PathVisio application, this plugin presents a simple to use and cross-platform application for the construction of complex MIM diagrams with the ability to annotate diagram elements with comments, literature references and links to external databases. This tool extends the capabilities of the PathVisio pathway editor by providing both MIM-specific glyphs and support for a MIM-specific markup language file format for exchange with other MIM-compatible tools and diagram validation.
AVAILABILITY: The PathVisio-MIM plugin is freely available and works with versions of PathVisio 2.0.11 and later on Windows, Mac OS X and Linux. Information about MIM notation and the MIMML format is available at http://discover.nci.nih.gov/mim. The plugin, along with diagram examples, instructions and Java source code, may be downloaded at http://discover.nci.nih.gov/mim/mim_pathvisio.html.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17160905">An evaluation of power and type I error of single-nucleotide polymorphism transmission/disequilibrium-based statistical methods under different family structures, missing parental data, and population stratification. American journal of human genetics (2007)</a></h4>

        <!--
        <p><a class="toggle" title="An evaluation of power and type I error of single-nucleotide polymorphism transmission/disequilibrium-based statistical methods under different family structures, missing parental data, and population stratification." href="#">(Show/Hide Details)</a></p>

        <div id="2" style="display: none">
            <p>Researchers conducting family-based association studies have a wide variety of transmission/disequilibrium (TD)-based methods to choose from, but few guidelines exist in the selection of a particular method to apply to available data. Using a simulation study design, we compared the power and type I error of eight popular TD-based methods under different family structures, frequencies of missing parental data, genetic models, and population stratifications. No method was uniformly most powerful under all conditions, but type I error was appropriate for nearly every test statistic under all conditions. Power varied widely across methods, with a 46.5% difference in power observed between the most powerful and the least powerful method when 50% of families consisted of an affected sib pair and one parent genotyped under an additive genetic model and a 35.2% difference when 50% of families consisted of a single affection-discordant sibling pair without parental genotypes available under an additive genetic model. Methods were generally robust to population stratification, although some slightly less so than others. The choice of a TD-based test statistic should be dependent on the predominant family structure ascertained, the frequency of missing parental genotypes, and the assumed genetic model.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/19668183">The Systems Biology Graphical Notation. Nature biotechnology (2009)</a></h4>

        <!--
        <p><a class="toggle" title="The Systems Biology Graphical Notation." href="#">(Show/Hide Details)</a></p>

        <div id="3" style="display: none">
            <p>Circuit diagrams and Unified Modeling Language diagrams are just two examples of standard visual languages that help accelerate work by promoting regularity, removing ambiguity and enabling software tool support for communication of complex information. Ironically, despite having one of the highest ratios of graphical to textual information, biology still lacks standard graphical notations. The recent deluge of biological knowledge makes addressing this deficit a pressing concern. Toward this goal, we present the Systems Biology Graphical Notation (SBGN), a visual language developed by a community of biochemists, modelers and computer scientists. SBGN consists of three complementary languages: process diagram, entity relationship diagram and activity flow diagram. Together they enable scientists to represent networks of biochemical interactions in a standard, unambiguous way. We believe that SBGN will foster efficient and accurate representation, visualization, storage, exchange and reuse of information on all kinds of biological knowledge, from gene regulation, to metabolism, to cellular signaling.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/20921115">Biological validation of increased schizophrenia risk with NRG1, ERBB4, and AKT1 epistasis via functional neuroimaging in healthy controls. Archives of general psychiatry (2010)</a></h4>

        <!--
        <p><a class="toggle" title="Biological validation of increased schizophrenia risk with NRG1, ERBB4, and AKT1 epistasis via functional neuroimaging in healthy controls." href="#">(Show/Hide Details)</a></p>

        <div id="4" style="display: none">
            <p>NRG1 is a schizophrenia candidate gene and plays an important role in brain development and neural function. Schizophrenia is a complex disorder, with etiology likely due to epistasis.
OBJECTIVE: To examine epistasis between NRG1 and selected N-methyl-d-aspartate-glutamate pathway partners implicated in its effects, including ERBB4, AKT1, DLG4, NOS1, and NOS1AP.
MAIN OUTCOME MEASURES: Epistatic association between single-nucleotide polymorphisms (SNPs) and case-control status; epistatic association between SNPs and the blood oxygen level-dependent physiological response during working memory measured by functional magnetic resonance imaging.
RESULTS: We observed interaction between NRG1 5' and 3' SNPs rs4560751 and rs3802160 (likelihood ratio test P = .00020) and schizophrenia, which was validated using functional magnetic resonance imaging of working memory in healthy controls; carriers of risk-associated genotypes showed inefficient processing in the dorsolateral prefrontal cortex (P = .015, familywise error corrected). We observed epistasis between NRG1 (rs10503929; Thr286/289/294Met) and its receptor ERBB4 (rs1026882; likelihood ratio test P = .035); a 3-way interaction with these 2 SNPs and AKT1 (rs2494734) was also observed (odds ratio, 27.13; 95% confidence interval, 3.30-223.03; likelihood ratio test P = .042). These same 2- and 3-way interactions were further biologically validated via functional magnetic resonance imaging: healthy individuals carrying risk genotypes for NRG1 and ERBB4, or these 2 together with AKT1, were disproportionately less efficient in dorsolateral prefrontal cortex processing. Lower-level interactions were not observed between NRG1 /ERBB4 and AKT1 in association or neuroimaging, consistent with biological evidence that NRG1 × ERBB4 interaction modulates downstream AKT1 signaling.
CONCLUSION: Our data suggest complex epistatic effects implicating an NRG1 molecular pathway in cognitive brain function and pathogenesis of schizophrenia.</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/20084519">Evidence of statistical epistasis between DISC1, CIT and NDEL1 impacting risk for schizophrenia: biological validation with functional neuroimaging. Human genetics (2010)</a></h4>

        <!--
        <p><a class="toggle" title="Evidence of statistical epistasis between DISC1, CIT and NDEL1 impacting risk for schizophrenia: biological validation with functional neuroimaging." href="#">(Show/Hide Details)</a></p>

        <div id="5" style="display: none">
            <p>The etiology of schizophrenia likely involves genetic interactions. DISC1, a promising candidate susceptibility gene, encodes a protein which interacts with many other proteins, including CIT, NDEL1, NDE1, FEZ1 and PAFAH1B1, some of which also have been associated with psychosis. We tested for epistasis between these genes in a schizophrenia case-control study using machine learning algorithms (MLAs: random forest, generalized boosted regression andMonteCarlo logic regression). Convergence of MLAs revealed a subset of seven SNPs that were subjected to 2-SNP interaction modeling using likelihood ratio tests for nested unconditional logistic regression models. Of the 7C2 = 21 interactions, four were significant at the α = 0.05 level: DISC1 rs1411771-CIT rs10744743 OR = 3.07 (1.37, 6.98) p = 0.007; CIT rs3847960-CIT rs203332 OR = 2.90 (1.45, 5.79) p = 0.003; CIT rs3847960-CIT rs440299 OR = 2.16 (1.04, 4.46) p = 0.038; one survived Bonferroni correction (NDEL1 rs4791707-CIT rs10744743 OR = 4.44 (2.22, 8.88) p = 0.00013). Three of four interactions were validated via functional magnetic resonance imaging (fMRI) in an independent sample of healthy controls; risk associated alleles at both SNPs predicted prefrontal cortical inefficiency during the N-back task, a schizophrenia-linked intermediate biological phenotype: rs3847960-rs440299; rs1411771-rs10744743, rs4791707-rs10744743 (SPM5 p < 0.05, corrected), although we were unable to statistically replicate the interactions in other clinical samples. Interestingly, the CIT SNPs are proximal to exons that encode theDISC1 interaction domain. In addition, the 3' UTR DISC1 rs1411771 is predicted to be an exonic splicing enhancer and the NDEL1 SNP is ~3,000 bp from the exon encoding the region of NDEL1 that interacts with the DISC1 protein, giving a plausible biological basis for epistasis signals validated by fMRI.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/20829833">The BioPAX community standard for pathway data sharing. Nature biotechnology (2010)</a></h4>

        <!--
        <p><a class="toggle" title="The BioPAX community standard for pathway data sharing." href="#">(Show/Hide Details)</a></p>

        <div id="7" style="display: none">
            <p>Biological Pathway Exchange (BioPAX) is a standard language to represent biological pathways at the molecular and cellular level and to facilitate the exchange of pathway data. The rapid growth of the volume of pathway data has spurred the development of databases and computational tools to aid interpretation; however, use of these data is hampered by the current fragmentation of pathway information across many databases with incompatible formats. BioPAX, which was created through a community process, solves this problem by making pathway data substantially easier to collect, index, interpret and share. BioPAX can represent metabolic and signaling pathways, molecular and genetic interactions and gene regulation networks. Using BioPAX, millions of interactions, organized into thousands of pathways, from many organisms are available from a growing number of databases. This large amount of pathway data in a computable form will support visualization, analysis and biological discovery.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26685306">PaxtoolsR: Pathway Analysis in R Using Pathway Commons. Bioinformatics (Oxford, England) (2015)</a></h4>

        <!--
        <p><a class="toggle" title="PaxtoolsR: Pathway Analysis in R Using Pathway Commons." href="#">(Show/Hide Details)</a></p>

        <div id="9" style="display: none">
            <p>PURPOSE: PaxtoolsR package enables access to pathway data represented in the BioPAX format and made available through the Pathway Commons webservice for users of the R language to aid in advanced pathway analyses. Features include the extraction, merging, and validation of pathway data represented in the BioPAX format. This package also provides novel pathway datasets and advanced querying features for R users through the Pathway Commons webservice allowing users to query, extract, and retrieve data and integrate this data with local BioPAX datasets.
AVAILABILITY: The PaxtoolsR package is compatible with versions of R 3.1.1 (and higher) on Windows, Mac OS X, and Linux using Bioconductor 3.0 and is available through the Bioconductor R package repository along with source code and a tutorial vignette describing common tasks, such as data visualization and gene set enrichment analysis. Source code and documentation are at http://www.bioconductor.org/packages/paxtoolsr. This plugin is free, open-source and licensed under the LGPL-3.
CONTACT: paxtools@cbio.mskcc.org or lunaa@cbio.mskcc.org.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/25213708">Using drug response data to identify molecular effectors, and molecular "omic" data to identify candidate drugs in cancer. Human genetics (2015)</a></h4>

        <!--
        <p><a class="toggle" title="Using drug response data to identify molecular effectors, and molecular "omic" data to identify candidate drugs in cancer." href="#">(Show/Hide Details)</a></p>

        <div id="11" style="display: none">
            <p>The current convergence of molecular and pharmacological data provides unprecedented opportunities to gain insights into the relationships between the two types of data. Multiple forms of large-scale molecular data, including but not limited to gene and microRNA transcript expression, DNA somatic and germline variations from next-generation DNA and RNA sequencing, and DNA copy number from array comparative genomic hybridization are all potentially informative when one attempts to recognize the panoply of potentially influential events both for cancer progression and therapeutic outcome. Concurrently, there has also been a substantial expansion of the pharmacological data being accrued in a systematic fashion. For cancer cell lines, the National Cancer Institute cell line panel (NCI-60), the Cancer Cell Line Encyclopedia (CCLE), and the collaborative Genomics of Drug Sensitivity in Cancer (GDSC) databases all provide subsets of these forms of data. For the patient-derived data, The Cancer Genome Atlas (TCGA) provides analogous forms of genomic information along with treatment histories. Integration of these data in turn relies on the fields of statistics and statistical learning. Multiple algorithmic approaches may be chosen, depending on the data being considered, and the nature of the question being asked. Combining these algorithms with prior biological knowledge, the results of molecular biological studies, and the consideration of genes as pathways or functional groups provides both the challenge and the potential of the field. The ultimate goal is to provide a paradigm shift in the way that drugs are selected to provide a more targeted and efficacious outcome for the patient.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/21586134">A formal MIM specification and tools for the common exchange of MIM diagrams: an XML-Based format, an API, and a validation method. BMC bioinformatics (2011)</a></h4>

        <!--
        <p><a class="toggle" title="A formal MIM specification and tools for the common exchange of MIM diagrams: an XML-Based format, an API, and a validation method." href="#">(Show/Hide Details)</a></p>

        <div id="12" style="display: none">
            <p>BACKGROUND: The Molecular Interaction Map (MIM) notation offers a standard set of symbols and rules on their usage for the depiction of cellular signaling network diagrams. Such diagrams are essential for disseminating biological information in a concise manner. A lack of software tools for the notation restricts wider usage of the notation. Development of software is facilitated by a more detailed specification regarding software requirements than has previously existed for the MIM notation.
RESULTS: A formal implementation of the MIM notation was developed based on a core set of previously defined glyphs. This implementation provides a detailed specification of the properties of the elements of the MIM notation. Building upon this specification, a machine-readable format is provided as a standardized mechanism for the storage and exchange of MIM diagrams. This new format is accompanied by a Java-based application programming interface to help software developers to integrate MIM support into software projects. A validation mechanism is also provided to determine whether MIM datasets are in accordance with syntax rules provided by the new specification.
CONCLUSIONS: The work presented here provides key foundational components to promote software development for the MIM notation. These components will speed up the development of interoperable tools supporting the MIM notation and will aid in the translation of data stored in MIM diagrams to other standardized formats. Several projects utilizing this implementation of the notation are outlined herein. The MIM specification is available as an additional file to this publication. Source code, libraries, documentation, and examples are available at http://discover.nci.nih.gov/mim.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/23547033">PathVisio-Faceted Search: an exploration tool for multi-dimensional navigation of large pathways. Bioinformatics (Oxford, England) (2013)</a></h4>

        <!--
        <p><a class="toggle" title="PathVisio-Faceted Search: an exploration tool for multi-dimensional navigation of large pathways." href="#">(Show/Hide Details)</a></p>

        <div id="13" style="display: none">
            <p>PURPOSE: The PathVisio-Faceted Search plugin helps users explore and understand complex pathways by overlaying experimental data and data from webservices, such as Ensembl BioMart, onto diagrams drawn using formalized notations in PathVisio. The plugin then provides a filtering mechanism, known as a faceted search, to find and highlight diagram nodes (e.g. genes and proteins) of interest based on imported data. The tool additionally provides a flexible scripting mechanism to handle complex queries.
AVAILABILITY: The PathVisio-Faceted Search plugin is compatible with PathVisio 3.0 and above. PathVisio is compatible with Windows, Mac OS X and Linux. The plugin, documentation, example diagrams and Groovy scripts are available at http://PathVisio.org/wiki/PathVisioFacetedSearchHelp. The plugin is free, open-source and licensed by the Apache 2.0 License.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26766592">An Integrated Metabolic Atlas of Clear Cell Renal Cell Carcinoma. Cancer cell (2016)</a></h4>

        <!--
        <p><a class="toggle" title="An Integrated Metabolic Atlas of Clear Cell Renal Cell Carcinoma." href="#">(Show/Hide Details)</a></p>

        <div id="14" style="display: none">
            <p>Dysregulated metabolism is a hallmark of cancer, manifested through alterations in metabolites. We performed metabolomic profiling on 138 matched clear cell renal cell carcinoma (ccRCC)/normal tissue pairs and found that ccRCC is characterized by broad shifts in central carbon metabolism, one-carbon metabolism, and antioxidant response. Tumor progression and metastasis were associated with metabolite increases in glutathione and cysteine/methionine metabolism pathways. We develop an analytic pipeline and visualization tool (metabolograms) to bridge the gap between TCGA transcriptomic profiling and our metabolomic data, which enables us to assemble an integrated pathway-level metabolic atlas and to demonstrate discordance between transcriptome and metabolome. Lastly, expression profiling was performed on a high-glutathione cluster, which corresponds to a poor-survival subgroup in the ccRCC TCGA cohort.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/22570691">Gene expression profiles of the NCI-60 human tumor cell lines define molecular interaction networks governing cell migration processes. PloS one (2012)</a></h4>

        <!--
        <p><a class="toggle" title="Gene expression profiles of the NCI-60 human tumor cell lines define molecular interaction networks governing cell migration processes." href="#">(Show/Hide Details)</a></p>

        <div id="15" style="display: none">
            <p>Although there is extensive information on gene expression and molecular interactions in various cell types, integrating those data in a functionally coherent manner remains challenging. This study explores the premise that genes whose expression at the mRNA level is correlated over diverse cell lines are likely to function together in a network of molecular interactions. We previously derived expression-correlated gene clusters from the database of the NCI-60 human tumor cell lines and associated each cluster with function categories of the Gene Ontology (GO) database. From a cluster rich in genes associated with GO categories related to cell migration, we extracted 15 genes that were highly cross-correlated; prominent among them were RRAS, AXL, ADAM9, FN14, and integrin-beta1. We then used those 15 genes as bait to identify other correlated genes in the NCI-60 database. A survey of current literature disclosed, not only that many of the expression-correlated genes engaged in molecular interactions related to migration, invasion, and metastasis, but that highly cross-correlated subsets of those genes engaged in specific cell migration processes. We assembled this information in molecular interaction maps (MIMs) that depict networks governing 3 cell migration processes: degradation of extracellular matrix, production of transient focal complexes at the leading edge of the cell, and retraction of the rear part of the cell. Also depicted are interactions controlling the release and effects of calcium ions, which may regulate migration in a spaciotemporal manner in the cell. The MIMs and associated text comprise a detailed and integrated summary of what is currently known or surmised about the role of the expression cross-correlated genes in molecular networks governing those processes.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/24360018">SIRT1/PARP1 crosstalk: connecting DNA damage and metabolism. Genome integrity (2013)</a></h4>

        <!--
        <p><a class="toggle" title="SIRT1/PARP1 crosstalk: connecting DNA damage and metabolism." href="#">(Show/Hide Details)</a></p>

        <div id="17" style="display: none">
            <p>An intricate network regulates the activities of SIRT1 and PARP1 proteins and continues to be uncovered. Both SIRT1 and PARP1 share a common co-factor nicotinamide adenine dinucleotide (NAD+) and several common substrates, including regulators of DNA damage response and circadian rhythms. We review this complex network using an interactive Molecular Interaction Map (MIM) to explore the interplay between these two proteins. Here we discuss how NAD + competition and post-transcriptional/translational feedback mechanisms create a regulatory network sensitive to environmental cues, such as genotoxic stress and metabolic states, and examine the role of those interactions in DNA repair and ultimately, cell fate decisions.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/22581176">Software support for SBGN maps: SBGN-ML and LibSBGN. Bioinformatics (Oxford, England) (2012)</a></h4>

        <!--
        <p><a class="toggle" title="Software support for SBGN maps: SBGN-ML and LibSBGN." href="#">(Show/Hide Details)</a></p>

        <div id="6" style="display: none">
            <p>MOTIVATION: LibSBGN is a software library for reading, writing and manipulating Systems Biology Graphical Notation (SBGN) maps stored using the recently developed SBGN-ML file format. The library (available in C++ and Java) makes it easy for developers to add SBGN support to their tools, whereas the file format facilitates the exchange of maps between compatible software applications. The library also supports validation of maps, which simplifies the task of ensuring compliance with the detailed SBGN specifications. With this effort we hope to increase the adoption of SBGN in bioinformatics tools, ultimately enabling more researchers to visualize biological knowledge in a precise and unambiguous manner.
AVAILABILITY AND IMPLEMENTATION: Milestone 2 was released in December 2011. Source code, example files and binaries are freely available under the terms of either the LGPL v2.1+ or Apache v2.0 open source licenses from http://libsbgn.sourceforge.net.
CONTACT: sbgn-libsbgn@lists.sourceforge.net.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26420780">The digital revolution in phenotyping. Briefings in bioinformatics (2015)</a></h4>

        <!--
        <p><a class="toggle" title="The digital revolution in phenotyping." href="#">(Show/Hide Details)</a></p>

        <div id="8" style="display: none">
            <p>Phenotypes have gained increased notoriety in the clinical and biological domain owing to their application in numerous areas such as the discovery of disease genes and drug targets, phylogenetics and pharmacogenomics. Phenotypes, defined as observable characteristics of organisms, can be seen as one of the bridges that lead to a translation of experimental findings into clinical applications and thereby support 'bench to bedside' efforts. However, to build this translational bridge, a common and universal understanding of phenotypes is required that goes beyond domain-specific definitions. To achieve this ambitious goal, a digital revolution is ongoing that enables the encoding of data in computer-readable formats and the data storage in specialized repositories, ready for integration, enabling translational research. While phenome research is an ongoing endeavor, the true potential hidden in the currently available data still needs to be unlocked, offering exciting opportunities for the forthcoming years. Here, we provide insights into the state-of-the-art in digital phenotyping, by means of representing, acquiring and analyzing phenotype data. In addition, we provide visions of this field for future research work that could enable better applications of phenotype data.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://bioinformatics.oxfordjournals.org/content/28/6/889.full.pdf+html">PathVisio-Validator: a rule-based validation plugin for graphical pathway notations Bioinformatics (2012)</a></h4>

        <!--
        <p><a class="toggle" title="PathVisio-Validator: a rule-based validation plugin for graphical pathway notations" href="#">(Show/Hide Details)</a></p>

        <div id="10" style="display: none">
            <p>The PathVisio-Validator plugin aims to simplify the task of producing biological pathway diagrams that follow graphical standardized notations, such as Molecular Interaction Maps or the Systems Biology Graphical Notation.</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/25758781">Alterations of DNA repair genes in the NCI-60 cell lines and their predictive value for anticancer drug activity. DNA repair (2015)</a></h4>

        <!--
        <p><a class="toggle" title="Alterations of DNA repair genes in the NCI-60 cell lines and their predictive value for anticancer drug activity." href="#">(Show/Hide Details)</a></p>

        <div id="16" style="display: none">
            <p>Loss of function of DNA repair (DNAR) genes is associated with genomic instability and cancer predisposition; it also makes cancer cells reliant on a reduced set of DNAR pathways to resist DNA-targeted therapy, which remains the core of the anticancer armamentarium. Because the landscape of DNAR defects across numerous types of cancers and its relation with drug activity have not been systematically examined, we took advantage of the unique drug and genomic databases of the US National Cancer Institute cancer cell lines (the NCI-60) to characterize 260 DNAR genes with respect to deleterious mutations and expression down-regulation; 169 genes exhibited a total of 549 function-affecting alterations, with 39 of them scoring as putative knockouts across 31 cell lines. Those mutations were compared to tumor samples from 12 studies of The Cancer Genome Atlas (TCGA) and The Cancer Cell Line Encyclopedia (CCLE). Based on this compendium of alterations, we determined which DNAR genomic alterations predicted drug response for 20,195 compounds present in the NCI-60 drug database. Among 242 DNA damaging agents, 202 showed associations with at least one DNAR genomic signature. In addition to SLFN11, the Fanconi anemia-scaffolding gene SLX4 (FANCP/BTBD12) stood out among the genes most significantly related with DNA synthesis and topoisomerase inhibitors. Depletion and complementation experiments validated the causal relationship between SLX4 defects and sensitivity to raltitrexed and cytarabine in addition to camptothecin. Therefore, we propose new rational uses for existing anticancer drugs based on a comprehensive analysis of DNAR genomic parameters.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/17234637">snp.plotter: an R-based SNP/haplotype association and linkage disequilibrium plotting package. Bioinformatics (Oxford, England) (2007)</a></h4>

        <!--
        <p><a class="toggle" title="snp.plotter: an R-based SNP/haplotype association and linkage disequilibrium plotting package." href="#">(Show/Hide Details)</a></p>

        <div id="18" style="display: none">
            <p>snp.plotter is a newly developed R package which produces high-quality plots of results from genetic association studies. The main features of the package include options to display a linkage disequilibrium (LD) plot below the P-value plot using either the r2 or D' LD metric, to set the X-axis to equal spacing or to use the physical map of markers, and to specify plot labels, colors, symbols and LD heatmap color scheme. snp.plotter can plot single SNP and/or haplotype data and simultaneously plot multiple sets of results. R is a free software environment for statistical computing and graphics available for most platforms. The proposed package provides a simple way to convey both association and LD information in a single appealing graphic for genetic association studies.
AVAILABILITY: Downloadable R package and example datasets are available at http://cbdb.nimh.nih.gov/~kristin/snp.plotter.html and http://www.r-project.org.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26020938">Predicted Role of NAD Utilization in the Control of Circadian Rhythms during DNA Damage Response. PLoS computational biology (2015)</a></h4>

        <!--
        <p><a class="toggle" title="Predicted Role of NAD Utilization in the Control of Circadian Rhythms during DNA Damage Response." href="#">(Show/Hide Details)</a></p>

        <div id="19" style="display: none">
            <p>The circadian clock is a set of regulatory steps that oscillate with a period of approximately 24 hours influencing many biological processes. These oscillations are robust to external stresses, and in the case of genotoxic stress (i.e. DNA damage), the circadian clock responds through phase shifting with primarily phase advancements. The effect of DNA damage on the circadian clock and the mechanism through which this effect operates remains to be thoroughly investigated. Here we build an in silico model to examine damage-induced circadian phase shifts by investigating a possible mechanism linking circadian rhythms to metabolism. The proposed model involves two DNA damage response proteins, SIRT1 and PARP1, that are each consumers of nicotinamide adenine dinucleotide (NAD), a metabolite involved in oxidation-reduction reactions and in ATP synthesis. This model builds on two key findings: 1) that SIRT1 (a protein deacetylase) is involved in both the positive (i.e. transcriptional activation) and negative (i.e. transcriptional repression) arms of the circadian regulation and 2) that PARP1 is a major consumer of NAD during the DNA damage response. In our simulations, we observe that increased PARP1 activity may be able to trigger SIRT1-induced circadian phase advancements by decreasing SIRT1 activity through competition for NAD supplies. We show how this competitive inhibition may operate through protein acetylation in conjunction with phosphorylation, consistent with reported observations. These findings suggest a possible mechanism through which multiple perturbations, each dominant during different points of the circadian cycle, may result in the phase advancement of the circadian clock seen during DNA damage.
</p>
        </div>
        -->
    </div>

    <div class="publication">
        <h4><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pubmed/26635141">rcellminer: exploring molecular profiles and drug response of the NCI-60 cell lines in R. Bioinformatics (Oxford, England) (2015)</a></h4>

        <!--
        <p><a class="toggle" title="rcellminer: exploring molecular profiles and drug response of the NCI-60 cell lines in R." href="#">(Show/Hide Details)</a></p>

        <div id="20" style="display: none">
            <p>PURPOSE: The rcellminer R package provides a wide range of functionality to help R users access and explore molecular profiling and drug response data for the NCI-60. The package enables flexible programmatic access to CellMiner's unparalleled breadth of NCI-60 data, including gene and protein expression, copy number, whole exome mutations, as well as activity data for ∼21K compounds, with information on their structure, mechanism of action and repeat screens. Functions are available to easily visualize compound structures, activity patterns and molecular feature profiles. Additionally, embedded R Shiny applications allow interactive data exploration.
AVAILABILITY AND IMPLEMENTATION: rcellminer is compatible with R 3.2 and above on Windows, Mac OS X and Linux. The package, documentation, tutorials and Shiny-based applications are available through Bioconductor (http://www.bioconductor.org/packages/rcellminer); ongoing updates will occur according to the Bioconductor release schedule with new CellMiner data. The package is free and open-source (LGPL 3).
CONTACT: lunaa@cbio.mskcc.org or vinodh.rajapakse@nih.gov.
</p>
        </div>
        -->
    </div>


    </div>
  </div>
</body>

</html>

